
Thu, 30 Jun 2016 10:07:22 GMT

COMMAND LINE:
/usr/bin/pypy pidigits.pypy-3.pypy 2000

PROGRAM FAILED 


PROGRAM OUTPUT:

Traceback (most recent call last):
  File "pidigits.pypy-3.pypy", line 10, in <module>
    from gmpy import mpz
ImportError: No module named gmpy
